packages/@orbit/record-cache/src/operators/async-query-operators.ts

```diff
@@ -174,6 +174,8 @@ function applyFilter(record: Record, filter: any) {
             actual !== undefined &&
             expected.some(e => actual.type === e.type && actual.id === e.id)
           );
+        } else if (expected === null) {
+          return !actual;
```

Comment:
Same comment as below:

```suggestion
          return actual === null;
```